test/e2e/tests/phishing-detection/phishing-detection.spec.js

```diff
@@ -1,12 +1,14 @@
 const { strict: assert } = require('assert');
+
```

Comment:
Thanks for moving this file into a phishing folder. One of the things I'm currently pushing is test organisation. Organising the tests by the primary Controller.